### ISSUE TITLE
diasble

### ISSUE THREAD

**Issue by ooooo-create** [ooooo-create/Public#1]
- inplace 算子都没有关闭，还没有找到单测位置

| num | opsname | is_same_operands_with_result | is_element_wise_binary |
| --- | --- | --- | --- |
| 1 | Abs | 1 |   |
| 2 | Abs_ | TODO |   |
| 3 | Acos | 1 |   |
| 4 | Acos_ | TODO |   |
| 5 | Acosh | 1 |   |
| 6 | Acosh_ | TODO |   |
| 7 | Angle | 1 |   |
| 8 | Asin | 1 |   |
| 9 | Asin_ | TODO |   |
| 10 | Asinh | 1 |   |
| 11 | Asinh_ | TODO |   |
| 12 | Atan | 1 |   |
| 13 | Atan_ | TODO |   |
| 14 | Atanh | 1 |   |
| 15 | Atanh_ | TODO |   |
| 16 | Hardtanh | 1 （别名算子 brelu） |   |
| 17 | Hardtanh_ | TODO |   |
| 18 | Bernoulli | TODO (找不到 check_output) |   |
| 19 | BitwiseNot | 1|   |
| 20 | BitwiseNot_ | TODO |   |
| 21 | Ceil | 1 |   |
| 22 | Ceil_ | TODO |   |
| 23 | Celu | TODO（check_output pir 没打开） |   |
| 24 | Clip | 1|   |
| 25 | Clip_ | TODO |   |
| 26 | Conj | 1|   |
| 27 | CopyTo | TODO（未找到单测） |   |
| 28 | Cos | 1|   |
| 29 | Cos_ | TODO |   |
| 30 | Cosh | 1|   |
| 31 | Cosh_ | TODO |   |
| 32 | DequantizeAbsMax | TODO( check_output 未打开 pir) |   |
| 33 | DequantizeLog | TODO（check_output 未打开 pir） |   |
| 34 | Digamma | 1|   |
| 35 | Digamma_ | TODO |   |
| 36 | Dirichlet | TODO (找不到 check_output)|   |
| 37 | DisableCheckModelNanInf | TODO (找不到单测)|   |
| 38 | Depend | TODO(找不到单测) |   |
| 39 | Elu | 1|   |
| 40 | Elu_ | TODO |   |
| 41 | EmptyLike | TODO(没有 optest 单测) |   |
| 42 | Erf | 1|   |
| 43 | Erf_ | TODO |   |
| 44 | Erfinv | 1|   |
| 45 | Erfinv_ | TODO |   |
| 46 | Exp | 1|   |
| 47 | Exp_ | TODO |   |
| 48 | Expm1 | 1|   |
| 49 | Expm1_ | TODO |   |
| 50 | Exponential_ | TODO |   |
| 51 | FakeDequantizeMaxAbs | TODO(check_output 未打卡 pir) |   |
| 52 | Fill | TODO(未打开 pir)fill_any |   |
| 53 | Fill_ | TODO |   |
| 54 | Fetch | TODO(找不到 optest 单测) |   |
| 55 | Flip | 1|   |
| 56 | Floor | 1|   |
| 57 | Floor_ | TODO |   |
| 58 | FullLike | 1|   |
| 59 | GetTensorFromSelectedRows | TODO （无 optest）|   |
| 60 | Gelu | 1|   |
| 61 | Gelu_ | TODO |   |
| 62 | Hardswish | TODO(找不到单测) |   |
| 63 | Imag | 1|   |
| 64 | Increment | TODO(没有 optest) |   |
| 65 | Increment_ | TODO |   |
| 66 | Isfinite | TODO (没有 optest)|   |
| 67 | IsfiniteSr | TODO (没有 optest)|   |
| 68 | Isinf | TODO(没有 optest) |   |
| 69 | IsinfSr | TODO(没有 optest) |   |
| 70 | Isnan | TODO(没有 optest) |   |
| 71 | IsnanSr | TODO(没有 optest) |   |
| 72 | I0 | 1|   |
| 73 | I0_ | TODO |   |
| 74 | I0e | 1|   |
| 75 | I1 | 1|   |
| 76 | I1e | 1|   |
| 77 | LabelSmooth | 1|   |
| 78 | Lgamma | 1 |   |
| 79 | Lgamma_ | TODO |   |
| 80 | Log1p | 1|   |
| 81 | Log1p_ | TODO |   |
| 82 | Log | 1|   |
| 83 | Log_ | TODO |   |
| 84 | Log10 | 1|   |
| 85 | Log10_ | TODO |   |
| 86 | Log2 | TODO (没有打开 PIR) |   |
| 87 | Log2_ | TODO |   |
| 88 | LogicalNot | TODO(没有 optest 单测) |   |
| 89 | LogicalNot_ | TODO |   |
| 90 | Logit | 1|   |
| 91 | Logit_ | TODO |   |
| 92 | Logsigmoid | TODO(chekouput 没有打开 pir) |   |
| 93 | Logsigmoid_ | TODO |   |
| 94 | Memcpy | TODO（没有 optest） |   |
| 95 | Mish | 1|   |
| 96 | NumberCount | TODO (没有打开 pir) |   |
| 97 | Pow | 1 |   |
| 98 | Poisson | TODO  (没有 check_ouput)|   |
| 99 | Pow_ | TODO |   |
| 100 | Prelu | 1|   |
| 101 | Print | TODO(没有 optest) |   |
| 102 | PutAlongAxis | 1|   |
| 103 | PutAlongAxis_ | TODO |   |
| 104 | Real | 1 |   |
| 105 | Reciprocal | 1 |   |
| 106 | Reciprocal_ | TODO |   |
| 107 | Relu | 1|   |
| 108 | Relu6 | 1|   |
| 109 | Relu_ | TODO |   |
| 110 | Reverse | TODO(没有打开 PIR) |   |
| 111 | Roll | 1 |   |
| 112 | Round | 1 |   |
| 113 | Round_ | TODO |   |
| 114 | RowConv | TODO (没有打开 pir) |   |
| 115 | Rsqrt | 1 |   |
| 116 | Rsqrt_ | TODO |   |
| 117 | ScaleSr | TODO (未找到单测) |   |
| 118 | ScaleSr_ | TODO |   |
| 119 | Scale_ | TODO |   |
| 120 | ScatterNdAdd | 1|   |
| 121 | Scatter | 1|   |
| 122 | Scatter_ | TODO |   |
| 123 | Select | TODO 未找到单测 [pr 结束点] |   |
| 124 | Selu | 1 |   |
| 125 | ShadowFeed | TODO （未找到单测）|   |
| 126 | ShareData_ | TODO |   |
| 127 | Sign | 1 |   |
| 128 | Sin | 1 |   |
| 129 | Sin_ | TODO |   |
| 130 | Sinh | 1 |   |
| 131 | Sinh_ | TODO |   |
| 132 | Slogdet | 1 |   |
| 133 | Softmax | 1 |   |
| 134 | Softmax_ | TODO |   |
| 135 | Softplus | 1 |   |
| 136 | SoftRelu | TODO（没有打开 pir） |   |
| 137 | Softshrink | 1 |   |
| 138 | Softsign | 1 |   |
| 139 | Stanh | 1 |   |
| 140 | Swish | TODO (没有打开 PIR) |   |
| 141 | Tan | 1|   |
| 142 | Tan_ | TODO |   |
| 143 | Tanh | 1|   |
| 144 | Tanh_ | TODO |   |
| 145 | Tril | 1 (未找到单测) test\legacy_test\test_tril_triu_op.py |   |
| 146 | Tril_ | TODO |   |
| 147 | Triu | 1|   |
| 148 | Triu_ | TODO |   |
| 149 | Trunc | 1 |   |
| 150 | Trunc_ | TODO |   |
| 151 | Sigmoid | 1 |   |
| 152 | Sigmoid_ | TODO |   |
| 153 | LeakyRelu | 1 |   |
| 154 | LeakyRelu_ | TODO |   |
| 155 | ThresholdedRelu | 1 |   |
| 156 | ThresholdedRelu_ | TODO |   |
| 157 | SquareSr | TODO |   |
| 158 | Square | 1 |   |
| 159 | Polygamma | 1 |   |
| 160 | Polygamma_ | TODO |   |
| 161 | EnableCheckModelNanInf | TODO(未找到) |   |
| 162 | ViewShape | TODO(未找到 ) |   |
| 163 | Silu | 1 |   |
| 164 | ViewDtype | TODO (未找到) |   |
| 165 | Sqrt | 1 |   |
| 166 | Sqrt_ | TODO |   |
| 167 | SqrtSr | 1 和 sqrtsr 一样？|   |
| 168 | SqrtSr_ | TODO |   |
| 169 | FusedSoftmaxMaskUpperTriangle | 1 |   |
| 170 | Gammaln | 1 |   |
| 171 | Gammaln_ | TODO |   |
| 172 | GaussianInplace | TODO (没有checkout) |   |
| 173 | GaussianInplace_ | TODO |   |
| 174 | Hardshrink | 1|   |
| 175 | HardSigmoid | 1|   |
| 176 | MergeSelectedRows | TODO (没有 optest) |   |
| 177 | NpuIdentity | TODO(没有找到单测) |   |
| 178 | Renorm | TODO(没有 optest 单测) |   |
| 179 | Renorm_ | TODO |   |
| 180 | TanhShrink | 1 |   |
| 181 | YoloBoxHead | TODO(没有找到单测) |   |
| 182 | StandardGamma | TODO(未找到单测)[pr 结束点] |   |
| 183 | Add_ |   | TODO |
| 183 | BitwiseAnd |   | 1 |
| 183 | BitwiseAnd_ |   | 1 |
| 183 | BitwiseRightShift |   | TODO |
| 183 | BitwiseRightShift_ |   | TODO |
| 183 | BitwiseLeftShift |   | TODO |
| 183 | BitwiseLeftShift_ |   | TODO |
| 183 | BitwiseOr |   | 1|
| 183 | BitwiseOr_ |   | TODO |
| 183 | BitwiseXor |   | 1 |
| 183 | BitwiseXor_ |   | TODO |
| 183 | Complex |   | 1 |
| 183 | Copysign |   | 1 |
| 183 | Copysign_ |   | TODO |
| 183 | Divide_ |   | TODO |
| 183 | ElementwisePow |   | 1 |
| 183 | Equal |   | TODO（未找到单测） |
| 183 | Equal_ |   | TODO |
| 183 | FloorDivide |   | 1 |
| 183 | Fmax |   | 1 |
| 183 | Fmin |   | 1 |
| 183 | Gammaincc |   | 1 |
| 183 | Gammaincc_ |   | TODO |
| 183 | GreaterEqual |   | TODO(未找到单测) |
| 183 | GreaterEqual_ |   | TODO |
| 183 | GreaterThan |   | TODO（未找到单测 |
| 183 | GreaterThan_ |   | TODO |
| 183 | Heaviside |   | 1 |
| 183 | LessEqual |   | TODO （未找到单测） |
| 183 | LessEqual_ |   | TODO |
| 183 | LessThan |   | TODO（未找到单测）|
| 183 | LessThan_ |   | TODO |
| 183 | LogicalAnd |   | TODO（没有 optest） |
| 183 | LogicalAnd_ |   | TODO |
| 183 | LogicalOr |   | TODO (没有 optest)|
| 183 | LogicalOr_ |   | TODO |
| 183 | LogicalXor |   | TODO （没有 optest ）|
| 183 | LogicalXor_ |   | TODO |
| 183 | Maximum |   | TODO(没有 optest ) |
| 183 | Minimum |   | TODO（没有 optest ） |
| 183 | MultiplySr |   | TODO(没有找到单测) |
| 183 | MultiplySr_ |   | TODO |
| 183 | Multiply_ |   | TODO |
| 183 | Nextafter |   | 1 |
| 183 | NotEqual |   | TODO (没有 optest) |
| 183 | NotEqual_ |   | TODO |
| 183 | Remainder |   | 1 |
| 183 | Remainder_ |   | TODO |
| 183 | Subtract_ |   | TODO |
